visivo/commands/init_phase.py

```diff
@@ -89,7 +89,6 @@ def init_phase():
         )
 
     if target_type == snowflake_type:
-        host = click.prompt("? Database host", type=str)
```

Comment:
Why is this gone?

Reply:
There's no host attribute in a snowflake connection. The `account` serves a similar purpose.